cou/steps/__init__.py

```diff
@@ -286,9 +309,24 @@ class UpgradeStep(BaseStep):
     prompt: bool = False
 
 
+class UnitUpgradeStep(UpgradeStep):
+    """Represents the upgrade step for an individual unit."""
+
+
 class PreUpgradeStep(UpgradeStep):
     """Represents the pre-upgrade step."""
 
 
 class PostUpgradeStep(UpgradeStep):
     """Represents the post-upgrade step."""
+
+
+def is_unit_upgrade_step(step: BaseStep) -> bool:
```

Comment:
Question: what is the intention for adding this check? I don't see this being used yet so was curious.

Reply:
Same question

Reply:
I used cherry-pick and forgot about it. I removed it.